phantompasswd.py

```diff
@@ -54,15 +60,64 @@ def load_manifest(self, domain):
         try:
             elt = self.driver.find_element_by_css_selector('link[rel="password-manifest"]')
             resp = requests.get(elt.get_attribute('href'), verify=not self.ignore_ssl_errors)
-            return json.loads(resp.content)
+            return self.load_data_from_content(resp.content)
         except NoSuchElementException:
             return {}
 
     def load_data(self, domain):
-        try:
-            return json.load(open(os.path.join('manifests', '%s.json' % domain)))
-        except IOError:
-            return self.load_manifest(domain)
+        manifests_path = self._get_manifests_path()
+        domain = self._get_safe_domain(domain)
+
+        for content_type in [self.JSON, self.YAML]:
+            try:
+                content = open(os.path.join(manifests_path, '{}.{}'.format(domain, content_type)))
+                return self.load_data_from_content(content, content_type)
+            except IOError:
+                # file unreadable (not found)
+                pass
```

Comment:
It's intentionally a passthrough. It tries the `.json` file, then the `.yaml` file, and if that doesn't exist, it goes on to try `<link …` in the HTML (and other methods that we haven't documented).